src/services/attendee/attendee-router.test.ts

```diff
@@ -0,0 +1,605 @@
+            Role.enum.USER
+        ).expect(StatusCodes.BAD_REQUEST);
+    });
+});
+
+describe("DELETE /attendee/favorites/:eventId", () => {
+    const eventId = uuidv4();
+
+    beforeEach(async () => {
+        await Database.ATTENDEE.deleteMany({});
+    });
+
+    it("should remove the event ID from the user's favorites", async () => {
+        await Database.ATTENDEE.create(
+            makeTestAttendee({ favorites: [eventId, "otherEvent"] })
+        );
+
+        await del(`/attendee/favorites/${eventId}`, Role.enum.USER).expect(
+            StatusCodes.OK
+        );
+
+        const updated = await Database.ATTENDEE.findOne({
+            userId: TESTER.userId,
+        });
+        expect(updated?.favorites).not.toContain(eventId);
+        expect(updated?.favorites).toContain("otherEvent");
+    });
+
+    it("should handle event ID not being in favorites", async () => {
+        await Database.ATTENDEE.create(
+            makeTestAttendee({ favorites: ["otherEvent"] })
+        );
+
+        await del(`/attendee/favorites/${eventId}`, Role.enum.USER).expect(
+            StatusCodes.OK
+        );
+
+        const updated = await Database.ATTENDEE.findOne({
+            userId: TESTER.userId,
+        });
+        expect(updated?.favorites).toEqual(["otherEvent"]);
+    });
+
+    it("should return 404 if attendee is not found", async () => {
+        const res = await del(
+            `/attendee/favorites/${eventId}`,
+            Role.enum.USER
+        ).expect(StatusCodes.NOT_FOUND);
+
+        expect(res.body).toEqual({ error: "UserNotFound" });
+    });
+
+    it("should return 401 if user is unauthenticated", async () => {
+        await del(`/attendee/favorites/${eventId}`).expect(
+            StatusCodes.UNAUTHORIZED
+        );
+    });
+
+    it("should return 403 if user does not have USER role", async () => {
+        await del(`/attendee/favorites/${eventId}`, Role.enum.STAFF).expect(
+            StatusCodes.FORBIDDEN
+        );
+    });
+
+    it("should return 400 for invalid event ID", async () => {
+        const invalidEventId = "this-is-not-a-valid-uuid";
+
+        await del(
+            `/attendee/favorites/${invalidEventId}`,
+            Role.enum.USER
+        ).expect(StatusCodes.BAD_REQUEST);
+    });
+});
+
+describe("GET /attendee/favorites", () => {
+    const uuidEvent = "123e4567-e89b-12d3-a456-426614174000";
+
+    beforeEach(async () => {
+        await Database.ATTENDEE.deleteMany({});
+    });
+
+    it("should return the attendee with their favorites", async () => {
+        const favorites = [uuidEvent, "f7bd25b0-c749-4900-a56e-4a680e4d79fb"];
+        await Database.ATTENDEE.create(makeTestAttendee({ favorites }));
+
+        const response = await get(
+            "/attendee/favorites",
+            Role.enum.USER
+        ).expect(StatusCodes.OK);
+
+        expect(response.body.userId).toBe(TESTER.userId);
+        expect(response.body.favorites).toEqual(favorites);
```

Comment:
Instead of doing a double expect, do a toMatchObject if you just want to check a few properties.
```suggestion
        expect(response.body).toMatchObject{
            userId: TESTER.userId
            favorites,
       }
```

This will also have a nicer diff if the test ever fails - so you don't fix one issue and then the other expect fails.